src/python/arcor2_arserver/globals.py

```diff
@@ -4,22 +4,19 @@
 
 from websockets.server import WebSocketServerProtocol as WsClient
 
-from arcor2.cached import UpdateableCachedProject, UpdateableCachedScene
 from arcor2.data import events
 from arcor2.logging import get_aiologger
 from arcor2.object_types.abstract import Generic
 from arcor2_arserver.lock import Lock
 from arcor2_arserver.object_types.data import ObjectTypeDict
+from arcor2_arserver.user import Users
 from arcor2_arserver_data.events.common import ShowMainScreen
 
 logger = get_aiologger("ARServer")
 VERBOSE: bool = False
 
 PORT: int = int(os.getenv("ARCOR2_SERVER_PORT", 6789))
 
-SCENE: Optional[UpdateableCachedScene] = None
-PROJECT: Optional[UpdateableCachedProject] = None
-
 MAIN_SCREEN: Optional[ShowMainScreen.Data] = ShowMainScreen.Data(ShowMainScreen.Data.WhatEnum.ScenesList)
 
 INTERFACES: Set[WsClient] = set()
```

Comment:
Nejdřív mě napadlo jestli toto není zbytečné, když máme třídu Users, ale vlastně není, protože ne všechny UI musí být zalogované.

Reply:
Tohle jsem přehlídl, zase je pravda, že to je na dvou místech... Motná bych to přesunul do třídy users a držel to jen jako UI beze jména